xtext/start.sh

```diff
@@ -1,6 +1,17 @@
-
 #! /bin/bash
 
+# clean any previous editor instances
+find ${CATALINA_HOME}/webapps/ -type f -not -name 'ROOT.war' -delete
+find ${CATALINA_HOME}/webapps/ -mindepth 1 -type d -not -name 'ROOT' -exec rm -rf {} \+
```

Comment:
Would this be a good place to also clean the `/editorserver/upload` and `editorserver/build` directories?

Reply:
Implemented in 3cd5039fb8edc135e499002946f8a9d5e69dca16